### PR TITLE
feat: Updating CNI & CNS to Support AKS Overlay on WS19

### PR DESCRIPTION
**Pull request by jaer-tsun**
**What type of PR is this?**

**What this PR does / why we need it**: Adding CNI & CNS update to support AKS Overlay on WS19.

**Which issue(s) this PR fixes**:

**Requirements**:
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [x] adds unit tests
- [x] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: vhdbuilder/packer/components.json [hunk 1]
```diff
@@ -64,8 +64,8 @@
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-cns:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.4.35",
```
Comment (tyler-lloyd) [inline vhdbuilder/packer/components.json:67]: we still have .35 as the version for Windows ... do we want to remove it here yet?

Comment (jaer-tsun) [reply vhdbuilder/packer/components.json:67]: I think this one is for linux, windows gets updated in vhdbuilder/packer/generate-windows-vhd-configuration.ps1

File: vhdbuilder/packer/generate-windows-vhd-configuration.ps1 [hunk 2]
```diff
@@ -90,7 +90,9 @@ if ($windowsSku -eq "2019") {
             "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.23.11", # for k8s 1.23.x
             "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.24.3", # for k8s 1.24.x
             # OMS-Agent (Azure monitor). Owner: ganga1980 (Ganga Mahesh Siddem)
-            "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-ciprod01182023-095c864a")
+            "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-ciprod01182023-095c864a",
+            # CNS (Container Networking Service). Owner: jaer-tsun (Jaeryn)
+            "mcr.microsoft.com/containernetworking/azure-cns:v1.4.44")
```
Comment (AbelHu) [inline vhdbuilder/packer/generate-windows-vhd-configuration.ps1:95]: Will azure-cns v1.4.44 be scheduled to WS2019 nodes? NOTE: In k8s 1.23, we still support users to create Windows agent pools with both docker and containerd.

Comment (jaer-tsun) [reply vhdbuilder/packer/generate-windows-vhd-configuration.ps1:95]: yup the new CNS will also deploy to windows 2019 for CNI overlay scenario

File: vhdbuilder/packer/generate-windows-vhd-configuration.ps1 [hunk 3]
```diff
@@ -99,6 +99,8 @@ if ($windowsSku -eq "2019") {
             "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.26.7", # for k8s 1.26.x
             # OMS-Agent (Azure monitor). Owner: ganga1980 (Ganga Mahesh Siddem)
             "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-3.1.5")
```
Comment (AbelHu) [inline vhdbuilder/packer/generate-windows-vhd-configuration.ps1:101]: Test failed. Need to remove `)`.

**Review by AbelHu — APPROVED** [review]

**Pull request merged**